mslib/msui/multilayers.py

```diff
@@ -311,6 +317,22 @@ def add_multilayer(self, name, wms):
             if self.cbMultilayering.isChecked():
                 widget.setCheckState(0, QtCore.Qt.Unchecked)
 
+            if self.is_linear:
```

Comment:
potentially add some client-defined "styles"...? The types of matplotlib yscale function would come to mind.

Reply:
The client can now choose between linear or log scale.